contracts/docs/plantuml/oethContracts.puml

```diff
@@ -27,7 +27,7 @@ object "OETH ARM" as arm <<Origin>><<Proxy>> #$originColor {
 }
 
 
-object "OETHDripper" as drip <<Origin>><<Proxy>> #$originColor {
+object "OETHFixedRateDripper" as drip <<Origin>><<Proxy>> #$originColor {
```

Comment:
Nick is gonna love that you updated the diagrams :)

Reply:
Ahah, but I didn't do it, he did it for me and asked me to cherry-pick it to my branch 😁

https://github.com/OriginProtocol/origin-dollar/pull/2363/commits/e11940dd17bd76c4d8a52b2c2cc04b5102d0fd54

Reply:
lol